projects/vdk-core/src/vdk/internal/core/config.py

```diff
@@ -112,7 +113,19 @@ def get_description(self, key: ConfigKey) -> str | None:
         :return: description
         """
         key = _normalize_config_key(key)
-        return self.__config_key_to_description.get(key)
+        desc = self.__config_key_to_description.get(key)
+        prefix = "SENSITIVE: " if self.is_sensitive(key) else ""
```

Comment:
This is the `get_description` method, so that prefix will be part of the description and not the key. It won't mess with the key order. Output looks like below (just an example).

<img width="1689" alt="Screenshot 2023-02-01 at 15 21 35" src="https://user-images.githubusercontent.com/91800778/216054932-50d1e90a-2caf-4c2d-8711-c2c70951c839.png">

Maybe I'm missing something, but this should be fine.

If you're also sorting by description somehow and want the description to say that it's sensitive at the end instead of using a prefix, we can do it no problem, but please clarify.

Reply:
Thanks for the screenshot. It helps. 

 I misread the code. I was thinking it's part of the key in the description. 
No need for capital letters. They are considered loud. 

Perhaps we can add something like this - add "the option is marked as sensitive" ? 
 

```
version_check_disabled                       Set to true if version check is disabled 
                                              completely. It might make sense for managed/cloud executions 
                                              where version is controlled.
                                              Default value is: 'False'
                                              The option is marked as sensitive.  

```

Reply:
Changed it to the way you suggested. 🙂